snuba/admin/clickhouse/system_queries.py

```diff
@@ -71,6 +83,14 @@ def is_query_select(sql_query: str) -> bool:
     return True if match else False
 
 
+def is_query_show(sql_query: str) -> bool:
+    """
+    Simple validation to ensure query is a show command
+    """
+    match = SHOW_QUERY_RE.match(sql_query)
+    return True if match else False
```

Comment:
nit
```suggestion
    return bool(SHOW_QUERY_RE.match(sql_query))
```

Reply:
Did it that way to match the style of the other "is_query" funcs. But your style is definitely more concise.